packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1329,6 +1330,28 @@ export default class NewVm extends BaseComponent {
 
   // DISKS -----------------------------------------------------------------------
 
+  _getSrsNotOnSameHost = createSelector(
+    () => this.state.state.existingDisks,
```

Comment:
`existingDisks` will not be updated in the interface instead of the selected `VDIs`.
Create a selector which returns a set containing the disks container. It will also reduce the complicity of the algorithme.

Reply:
At least, i think the set is not needed, the selector will return `true` if it have an incoherence between   disks or the disk container. Then in this case we need to show different message because the issue can be in the template disks or in the selected VDIs.

Reply:
`existingDisks` can be updated  ([u can see it here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/new-vm/index.js#L1348ToL1361))

Reply:
True :+1: . If we select an SR for `existing disks` it will impact the `VDIs` disks?

Reply:
The idea is each time the `existingDisks` change,  we will do unnecessarily loop to the `VDIs`.